cfgov/jinja2/v1/_includes/molecules/hero.html

```diff
@@ -32,45 +23,84 @@
 
    value.links[i].text:    A string for the text of the link.
 
-   value.is_button:        A boolean to change the CTA link to a button.
+   value.is_button:        A boolean to change the CTA link(s) to a button.
```

Comment:
Probably not, but I don't think there's a practical way to prevent the possibility.